### PR TITLE
Modify some media source tests and add a new test for MMR

### PR DESCRIPTION
**Pull request by zhonghaishan**
Modify some media source tests
and also add a new MMR scenario test: add source url to child node

### CODE REVIEW

File: media-source/mediasource-play-hzhong.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```
Comment (dcarpente) [inline media-source/mediasource-play-hzhong.html]: Change the name of this file to reflect the test case (e.g. mediasource-sourcenode.html)

**Comment by dcarpente** [comment]
As discussed offline, let's move these modified tests to mmr-testing

**Comment by zhonghaishan** [comment]
> As discussed offline, let's move these modified tests to mmr-testing

I will create a different PR for copying tests to MMR-testing folder.
This one is for code review

**Comment by zhonghaishan** [comment]
We will not merge this PR. 
This will be just for comparison.

**Pull request closed without merging**